### PR TITLE
 BLOCKCHAIN-478 - Add identities for gov/technical/external accounts 

### PR DESCRIPTION
**Pull request by kacperzuk-neti**

### CODE REVIEW

File: substrate/bin/node/runtime/src/migrations.rs [hunk 1]
```diff
@@ -37,71 +70,3 @@ pub mod add_contracts_registry_pallet {
 		}
 	}
 }
-
-pub mod add_sora_bridge {
```
Comment (DorianSternVukotic) [inline substrate/bin/node/runtime/src/migrations.rs:41]: why delete this ?

File: substrate/bin/node/runtime/src/migrations.rs [hunk 2]
```diff
@@ -21,14 +24,44 @@ pub mod add_contracts_registry_pallet {
...
+				("5EYCAe5hveooUENA5d7dwq3caqM4LLBzktNumMKmhNRXu4JE", b"Senate".to_vec()),
+				(
+					"5EYCAe5iXF2YZuVZv1vig4xvf1CcDVocZCWYrv3TVSXpMTYA",
+					b"Citizenship Office".to_vec(),
+				),
+				//("5EYCAe5ijiYfyeZ2JJCGq56LmPyNRAKzpG4QkoQkkQNB5e6Z", b"Treasury"), - skipping this as Polkadot.js Apps have it builtin
+				("5EYCAe5g8CDuMsTief7QBxfvzDFEfws6ueXTUhsbx5V81nGH", b"Congress".to_vec()),
+				(
+					"5CSxW2nn4mQckisBziai4wGqQLEdKTUA4Xnt8iy8jtj5q52Q",
+					b"Onboarder LLD Faucet".to_vec(),
```
Comment (DorianSternVukotic) [inline substrate/bin/node/runtime/src/migrations.rs:38]: onboarder faucet has the identity already, skip it

Add the offices though, company registry office, land and metaverse land offices, and the upcoming ministry of finance office

File: substrate/bin/node/runtime/src/migrations.rs (hunk 1 above)
Comment (kacperzuk-neti) [reply substrate/bin/node/runtime/src/migrations.rs:41]: It's a migration, so it was needed only once for single runtime upgrade (v18 -> v19). It's no longer used anywhere, no reason to keep it.

File: substrate/bin/node/runtime/src/migrations.rs (hunk 2 above)
Comment (kacperzuk-neti) [reply substrate/bin/node/runtime/src/migrations.rs]: Done.

File: substrate/bin/node/runtime/src/migrations.rs [hunk 3]
```diff
@@ -21,14 +24,47 @@ pub mod add_contracts_registry_pallet {
...
+				("5GmkwXZ94cuLMMbtE5VtBaLpFDehoEpy6MZnJhkCSicxePs2", b"SORA Bridge".to_vec()),
+				("5DSfG3S7qSZzrDMj3F3qYybXAy1BLsVpRG5CNwRdwwNPjgVm", b"MEXC".to_vec()),
+				("5HEX1wk33NHAeEJV3B6goDHMJTqhy411znCUmfEAxKkQeqds", b"Coinstore".to_vec()),
+				("5EYCAe5iXF2YZiuxDWAcwtPMDaG7ihsYzPxajcNKRpNyD1Zi", b"Company Registry Office".to_vec()),
+				("5EYCAe5iXF2YZzoQHqGhj9dtcsUNB4puM5GqR1BwVHZyaWxM", b"Land Registry Office".to_vec()),
+				("5EYCAe5iXF2Ya2c2iKjeFWUAXAEcMyoKCBPvhRy8YprHTLNd", b"Metaverse Land Registry Office".to_vec()),
+				("5EYCAe5iXF2YZfPZ4arKGSYZvQruDacGHkuw4qAsQSQsKpMK", b"Asset Registry Office".to_vec()),
+				("5EYCAe5iXF2YZpCZr7ALYUUYaNpMXde3NUXxYn1Sc1YRM4gV", b"Ministry of Finance Office".to_vec()),
+				("5FBQRNJfzsttYvw1XnSwxwSUmb7A3EYm4q8aiscADREqvzYz", b"Wrapped LLD Token Contract".to_vec()),
+				("5EYCAe5ijGqt3WEM9aKUBdth51NEBNz9P84NaUMWZazzWt7c", b"LLM Politipool Technical Account".to_vec()),
```
Comment (kacperzuk-neti) [inline substrate/bin/node/runtime/src/migrations.rs:45]: @DorianSternVukotic Added a new one, politipool. Please check if the description is OK.

File: substrate/frame/identity/src/lib.rs [hunk 4]
```diff
@@ -998,4 +998,23 @@ impl<T: Config> Pallet<T> {
 		IdentityOf::<T>::get(who)
 			.map_or(false, |registration| (registration.info.fields().0.bits() & fields) == fields)
 	}
+
+	/// Set identity for some account
+	pub fn set_identity_no_deposit(
```
Comment (DorianSternVukotic) [inline substrate/frame/identity/src/lib.rs:1003]: Is there really no way to do a migration without editing identity pallet with a new function?
How callable is this for users ?

Comment (kacperzuk-neti) [reply substrate/frame/identity/src/lib.rs:1003]: > Is there really no way to do a migration without editing identity pallet with a new function?

There is. We could define a storage alias for the `IdentityOf` in the migration and manipulate it directly. This way was just easier and I don't see a good reason to not extend identity pallet like this. Especially that it's a temporary extension - we can remove this function together with the migration after it's executed on all live networks.

> How callable is this for users ?

It's not. It's only available from inside the runtime, it's not a pallet call.

**Pull request merged**